src/store/modules/ui.store.js

```diff
@@ -77,12 +84,13 @@ export default {
          */
         mode: UIModes.PHONE, // Configured in screen-size-management.plugin.js (or manually in the settings)
         /**
-         * Flag telling if the tooltip should be displayed over the map, floating and positioned at
-         * the feature's coordinates. If false, the tooltip will be displayed in the footer
+         * Expected position of the features tooltip position when selecting features.
          *
-         * @type Boolean
+         * The default position depends on the UI Modes and is checked on the fly
+         *
+         * @type String
          */
-        floatingTooltip: false, // Configured in screen-size-management.plugin.js
+        featureInfoPosition: FeatureInfoPositions.DEFAULT,
```

Comment:
See comment about we need to define the default behavior in refinement meeting, which means that if the default needs to change it also needs to change here, to be discussed.